### ISSUE TITLE
hello, what is the ratio of training set and validation set?

### ISSUE THREAD

**Issue by meiling-fdu** [parwal-lp/De-Fake_nn_final_project#1]

**Comment by parwal-lp** [comment]
It's 50/50 for all the models.

More specifically:
- Detector: 
-- training dataset has 100 images (50 real and 50 fake)
-- testing is done with three different datasets, each one with 100 images (50 real and 50 fake)

- Attributor:
-- both training and testing datasets have 200 images (50 per each class)
